redis/src/main/java/org/hibernate/ogm/datastore/redis/RedisDialect.java

```diff
@@ -83,7 +84,8 @@ public GridType overrideType(Type type) {
 
 	@Override
 	public Tuple getTuple(EntityKey key, TupleContext tupleContext) {
-		Entity entity = getEntity( key );
+		Entity entity = entityStorageStrategy.getEntity( entityId( key ) );
```

Comment:
Adding Id's back to the entity is only necessary for bulk access such as `getTuples` and `forEachTuple`?